public/app/features/dashboard/components/DashboardSettings/test-utils.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
This might seem like a lot of boilerplate for one test, but it can be used anywhere we test a component that uses `useDispatch`, `useSelector`, etc. I'll probably move the file out of the `DashboardSettings` folder next time I refactor a component.